problems/2021-cnrs-benchmark/phase-2/README.MD

```diff
@@ -6,13 +6,14 @@ Blue Waters supercomputer with 512 MPI processes on 32 XK or 16
 XE computing nodes. We recommend running these time dependent simulations on
 clusters with at least 64 independent processors and 64 GB of RAM. The
 transient case with the shortest perturbation period
-(`transient-perturb-7.i`) took ~24h to complete
-5000 timesteps and reach reasonable convergence of the gain and phase shift
-values on the 512 MPI processes on Blue Water. `transient-perturb-1.i` took
-~125h to complete 4000 timesteps under the same conditions.
+took ~0.67h to complete one period (200 timesteps) and 16.67h in total (5000
+timesteps) to reach reasonable convergence of the gain and phase shift
+values on the 512 MPI processes on Blue Water. Conversely, the case with the
+longest perturbation period took x2.25 longer due to the longer simulation time
+needed to reach convergence.
```

Comment:
Is this update from the most recent paper, [2]?

Reply:
Yea, the old compute times were based on preliminary results.